app/_landing_pages/insomnia/security.yaml

```diff
@@ -1,5 +1,10 @@
 metadata:
   title: Security at Insomnia
+  description: "Learn how Insomnia prioritizes data protection through encryption, product and application security, and organizational measures."
```

Comment:
Is this what I'm learning on this page?

Reply:
I think we need to split out security into just being about the legal thing and we can handle auth and authorization in another landing page.